dot/state/grandpa.go

```diff
@@ -7,13 +7,23 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"sync"
 
 	"github.com/ChainSafe/chaindb"
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
+var (
+	errPendingScheduledChanges = errors.New("pending scheduled changes needs to be applied")
+	errDuplicateHashes         = errors.New("duplicated hashes")
+	errAlreadyHasForcedChange  = errors.New("already has a forced change")
+	errUnfinalizedAncestor     = errors.New("unfinalized ancestor")
```

Comment:
any reason why these are unexported?

Reply:
Not being used anywhere, it is just as an error sentinel/signalization